ethereum/core/src/main/java/org/hyperledger/besu/ethereum/bonsai/BonsaiSnapshotWorldStateKeyValueStorage.java

```diff
@@ -67,6 +67,14 @@ public BonsaiUpdater updater() {
         trieLogStorage);
   }
 
+  @Override
+  public void close() throws Exception {
+    accountStorage.close();
```

Comment:
we don't close trieLogStorage ?

Reply:
trie log storage is not based on a snapshot, so we don't have to worry about closing it.